composer.required.json

```diff
@@ -45,11 +46,6 @@
     "enable-patching": true,
     "patchLevel": {
       "drupal/core": "-p2"
-    },
-    "patches": {
-      "drupal/core": {
```

Comment:
@grasmash i'm not sure that this patch should come out, i don't think this issue was actually fixed until 8.6.x (see https://www.drupal.org/node/2752961)